engine/src/main/java/org/hibernate/validator/internal/engine/ValidatorFactoryImpl.java

```diff
@@ -678,4 +625,202 @@ public String toString() {
+
+				this.messageInterpolator = defaultContext.messageInterpolator;
+				this.traversableResolver = defaultContext.traversableResolver;
+				this.parameterNameProvider = defaultContext.parameterNameProvider;
+				this.clockProvider = defaultContext.clockProvider;
+				this.scriptEvaluatorFactory = defaultContext.scriptEvaluatorFactory;
+				this.temporalValidationTolerance = defaultContext.temporalValidationTolerance;
+				this.failFast = defaultContext.failFast;
+				this.traversableResolverResultCacheEnabled = defaultContext.traversableResolverResultCacheEnabled;
+			}
+
+			public Builder setMessageInterpolator(MessageInterpolator messageInterpolator) {
+				if ( messageInterpolator == null ) {
+					this.messageInterpolator = defaultContext.messageInterpolator;
+				}
+				else {
+					this.messageInterpolator = messageInterpolator;
+				}
+
+				return this;
+			}
+
+			public Builder setTraversableResolver(TraversableResolver traversableResolver) {
+				if ( traversableResolver == null ) {
+					this.traversableResolver = defaultContext.traversableResolver;
+				}
+				else {
+					this.traversableResolver = traversableResolver;
+				}
+				return this;
+			}
+
+			public Builder setParameterNameProvider(ParameterNameProvider parameterNameProvider) {
+				if ( parameterNameProvider == null ) {
+					this.parameterNameProvider = defaultContext.parameterNameProvider;
+				}
+				else {
+					this.parameterNameProvider = new ExecutableParameterNameProvider( parameterNameProvider );
+				}
+				return this;
+			}
+
+			public Builder setClockProvider(ClockProvider clockProvider) {
+				if ( clockProvider == null ) {
+					this.clockProvider = defaultContext.clockProvider;
+				}
+				else {
+					this.clockProvider = clockProvider;
+				}
+				return this;
+			}
+
+			public Builder setTemporalValidationTolerance(Duration temporalValidationTolerance) {
+				this.temporalValidationTolerance = temporalValidationTolerance == null ? Duration.ZERO : temporalValidationTolerance.abs();
+				return this;
+			}
+
+			public Builder setScriptEvaluatorFactory(ScriptEvaluatorFactory scriptEvaluatorFactory) {
+				if ( scriptEvaluatorFactory == null ) {
+					this.scriptEvaluatorFactory = defaultContext.scriptEvaluatorFactory;
+				}
+				else {
+					this.scriptEvaluatorFactory = scriptEvaluatorFactory;
+				}
+				return this;
+			}
+
+			public Builder setFailFast(boolean failFast) {
+				this.failFast = failFast;
+				return this;
+			}
+
+			public Builder setTraversableResolverResultCacheEnabled(boolean traversableResolverResultCacheEnabled) {
+				this.traversableResolverResultCacheEnabled = traversableResolverResultCacheEnabled;
+				return this;
+			}
+
+			public ValidatorFactoryContext build() {
+				return new ValidatorFactoryContext( messageInterpolator, traversableResolver, parameterNameProvider, clockProvider, temporalValidationTolerance, scriptEvaluatorFactory, failFast, traversableResolverResultCacheEnabled );
```

Comment:
I would be surprised if it's all on a 120 characters line :).

A local ctrl+shift+f will help.